truss/templates/trtllm-briton/src/extension.py

```diff
@@ -33,7 +36,11 @@ class Extension:
     """
 
     def __init__(self, *args, **kwargs):
-        self._model = Model(*args, **kwargs)
+        self._config = kwargs["config"]
+        if TRTLLM_SPEC_DEC_TARGET_MODEL_NAME in self._config.get("trt_llm"):
```

Comment:
This seems a little weird. We should have a more clear way of indicating spec dec.